core/src/display_object.rs

```diff
@@ -1522,6 +1522,11 @@ pub trait TDisplayObject<'gc>:
     fn as_stage(&self) -> Option<Stage<'gc>> {
         None
     }
+    fn state(&self) -> Option<ButtonState> {
```

Comment:
nit: These functions should probably be placed higher up (around line 1322).

Reply:
Fixed now, Thank you :)